sqlglotrs/src/token.rs

```diff
@@ -0,0 +1,428 @@
+use pyo3::prelude::*;
+use strum_macros::EnumString;
+
+#[allow(non_camel_case_types, dead_code)]
+#[derive(Copy, Clone, Debug, Hash, PartialEq, Eq, EnumString)]
+#[pyclass]
+pub enum TokenType {
```

Comment:
is this auto generated?

Reply:
Not yet. Going to add this in follow-up PRs. It won't be checked into the repo at that point.